apl-api/src/main/java/com/apollocurrency/aplwallet/api/request/DetectMimeTypeUploadForm.java

```diff
@@ -0,0 +1,29 @@
+/*
+ * Copyright © 2018-2020 Apollo Foundation
+ */
+
+package com.apollocurrency.aplwallet.api.request;
+
+import javax.ws.rs.FormParam;
+
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.NoArgsConstructor;
+import org.jboss.resteasy.annotations.providers.multipart.PartType;
+
+/**
+ * Form data submitted to detect mime-type for file or 'data' field content
+ * That class is used for OpenAPI displaying mostly. DO NOT REMOVE !!
+ */
+@NoArgsConstructor
+public class DetectMimeTypeUploadForm {
+
+    @FormParam("file")
+    @PartType("charset=utf-8")
+    @Schema(type = "string", format = "binary", description = "Uploaded file name, optional, limited by Max Upload Size (42 Kb by default)")
```

Comment:
File itself or fileName?

Reply:
Yes, 'file' field for file content sent to server.

Reply:
Description 'Uploaded file name, optional,' -> 'Uploaded file (content), optional'

Reply:
Done